jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -72,7 +72,7 @@
               export PROJECT="k8s-jkns-e2e-gce-federation"
               export FEDERATION=true
               export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
-              export FEDERATIONS_DOMAIN_MAP="federation=ubernetes.io"
+              export FEDERATIONS_DOMAIN_MAP="federation=k8s-federation.com"
```

Comment:
It doesn't hurt to add trailing dot here as well, but we made `KubeDNS` to handle this, so it's fine.